src/platform/forms-system/test/js/helpers.unit.spec.js

```diff
@@ -1003,4 +1005,98 @@ describe('Schemaform helpers:', () => {
       expect(omitRequired(schema)).to.eql(expected);
     });
   });
+
+  describe('getNumeratedUiSchema', () => {
+    const uiSchema = {
+      'ui:title': 'My form',
+      'ui:options': {
+        someValue: true,
+      },
+      firstName: {
+        'ui:title': 'First Name',
+      },
+      lastName: {
+        'ui:title': 'Last Name',
+      },
+    };
+
+    it('should numerate the keys for uiSchema fields', () => {
+      const numeratedUiSchema = getNumeratedUiSchema(uiSchema, 4);
```

Comment:
`4` is an arbitrary `index`, correct?

Reply:
Yup, it's arbitrary. When this function gets called in a `map` callback though, each successive call will have an index incremented by 1.